content/fr/docs/reference/kubectl/quick-reference.md

```diff
@@ -0,0 +1,504 @@
+---
+title: Référence rapide de kubectl
```

Comment:
```suggestion
title: Prise en main de kubectl
```

Reply:
What do you think ? `Référence rapide de kubectl` is not French...

Reply:
i agree, it's correct but not too french,  since we're talking about a cheatsheet , what about "Aide-mémoire de kubectl"  ?

Reply:
> i agree, it's correct but not too french,  since we're talking about a cheatsheet , what about "Aide-mémoire de kubectl"  ? 

Yes, it's better !

Reply:
great , change made !